### PR TITLE
build(ci): reduce number of jobs

### PR DESCRIPTION
**Pull request by priyajeet**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: .travis.yml [hunk 1]
```diff
@@ -13,17 +13,11 @@ before_install:
...
-      script: npm-run-all clean build:i18n && echo "Running Flow on $(yarn flow ls | wc -l) files" && yarn flow
-    - name: "Unit Tests"
-      script: yarn test
+    - name: "Lint"
+      script: commitlint-travis && npm-run-all clean build:i18n lint && echo "Running Flow on $(yarn flow ls | wc -l) files" && yarn flow
     - name: "E2E Tests"
       script: yarn test:e2e
-    - name: "Build Babel"
-      script: npm-run-all clean build:i18n build:ci:es
+    - name: "Unit Tests && Build Babel"
```
Comment (jstoffan) [inline .travis.yml]: How about something like this:

```
    - name: "Lint"
      script: commitlint-travis && npm-run-all clean build:i18n lint && echo "Running Flow on $(yarn flow ls | wc -l) files" && yarn flow
    - name: "Tests"
      script: npm-run-all test test:e2e
    - name: "Build"
      script: npm-run-all clean build:i18n build:ci:es build:ci:dist
```

Comment (priyajeet) [reply .travis.yml]: I moved the babel build next to lint, since that one is fast.
e2e dist and tests are the 3 longest running, so take the most time.

Comment (priyajeet) [reply .travis.yml]: I now combined linting and build in 1 step since it was still fast, unit and e2e. So 3 jobs now.

**Pull request merged**